### PR TITLE
Misc quickstart fixes

### PR DESCRIPTION
**Pull request by jpvajda**
This PR fixes numerous problems with quickstarts including:

- misc typos and line formatting issues
- titles that should be sentence case
- removes "quickstarts" from titles as it's redundant

### CODE REVIEW

File: quickstarts/python/webpy/config.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 id: 535f37ab-3b88-471d-b107-266f39d28819
 name: webpy
+title: webpy
+
+# webpy should be lowercase.
+
 description: |
   ## Why monitor Webpy?
```
Comment (josephgregoryii) [inline quickstarts/python/webpy/config.yml:8]:
```suggestion
  ## Why monitor webpy?
```

File: quickstarts/python/pyramid/config.yml [hunk 2]
```diff
@@ -1,35 +1,42 @@
 id: 02f1dc20-9034-405b-90c1-8ad1bbf7755e
 name: pyramid
+title: pyramid
```
[the remark this answers is not in the document]
Comment (jpvajda) [reply quickstarts/python/pyramid/config.yml:3]: That looks like it's capitalized I'll fix that as well

https://trypyramid.com/

File: quickstarts/python/web2py/config.yml [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+
+# web2py should be lowercase.
+
 description: |
   ## What is web2py?
 
-  Open source, Python-based web application framework that enables development
-  via browser.
-
+  Web2py is an open-source web application framework and allows web developers to program dynamic web content using Python. Web2py is designed to help reduce tedious web development tasks, such as developing web forms from scratch, although a web developer may build a form from scratch if required.
```
[the remark this answers is not in the document]
Comment (jpvajda) [reply quickstarts/python/web2py/config.yml:10]: Will fix those. good catch

File: quickstarts/python/webpy/config.yml (hunk 1 above)
Comment (jpvajda) [reply quickstarts/python/webpy/config.yml:8]: wil fix. thanks!

**Comment by jpvajda** [comment]
@josephgregoryii I resolved all the feedback, could you do another review? 🙏

**Review by josephgregoryii — APPROVED** [review]
:shipit:

**Pull request merged**